src/main/java/org/opensearch/flowframework/util/ParseUtils.java

```diff
@@ -385,4 +401,17 @@ public static String parseArbitraryStringToObjectMapToString(Map<String, Object>
         String mappedString = mapper.writeValueAsString(map);
         return mappedString;
     }
+
+    /**
+     * Generates a String to String map based on a Json File
+     * @param path file path
+     * @return instance of the string
+     * @throws JsonProcessingException JsonProcessingException from Jackson for issues processing map
+     */
+    public static Map<String, String> parseJsonFileToStringToStringMap(String path) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        String jsonContent = resourceToString(path);
+        Map<String, String> mappedJsonFile = mapper.readValue(jsonContent, Map.class);
```

Comment:
curiosity question: is all values in the JSON file are text? Asking because if the JSON contains nested objects or arrays, this `Map<String, String>` might fail

Reply:
Yeah right now this method is only used for our defaults file which always will be Map<String, String>.In 2.14 we might find ways for customers to add there own, in which we will have to do stricter checking.